evaluate/evaluate_test.go

```diff
@@ -180,6 +180,8 @@ func TestEvaluate(t *testing.T) {
 			Before: func(t *testing.T, logger *log.Logger, resultPath string) {
 				// Set up mocks, when test is running.
 				mockedModel.MockCapabilityWriteTests.On("WriteTests", mock.Anything).Return(nil, ErrEmptyResponseFromModel)
+				mockedModel.MockModel.On("Name").Return("Empty Response Model")
+				mockedModel.MockModel.On("Cost").Return(0.0001)
```

Comment:
please put that in the helper that creates the mock since we will have it all the time